Update struct.md (#28893)
dotnet/docs: docs/csharp/language-reference/builtin-types/struct.md

  :::code language="csharp" source="snippets/shared/StructType.cs" id="ReadonlyWithInit":::

> You can apply the `readonly` modifier to static members of a structure type excluding static members able to modify the structure instance.

Reviewer: Thanks for adding your comments @chessboards. That helps. I'd suggest updating this PR as follows:

```suggestion
You can apply the `readonly` modifier to static fields of a structure type, but not any other static members, such as properties or methods.
```
